__test__/app.test.js

```diff
@@ -20,5 +20,89 @@ describe('/api', () => {
   //Joe BRANCH OUT FOR EACH REQUEST!! DON'T WORK ON MASTER
   describe('/tables', () => {});
   // Zak BRANCH OUT FOR EACH REQUEST!! DON'T WORK ON MASTER
-  describe('/users', () => {});
+  describe('/users', () => {
+    describe('GET', () => {
+      it('SUCCESS - Status 200 - responds with an array of all users', () => {
+        return request(app)
+          .get('/api/users')
+          .expect(200)
+          .then(({ body: { users } }) => {
+            expect(users).toEqual([
+              {
+                email: 'harryp@gmail.com',
```

Comment:
I would suggest using objectContaining in the forEach to avoid copying in all the data for users.. have a look in get tables